.github/workflows/build.yml

```diff
@@ -37,19 +37,61 @@ jobs:
           python setup.py bdist_wheel
 
       - name: Run Tests
-        run : pipenv run tests # execute Pipfile script tests
-
-      - name: Build Docs
-        run: pipenv run docs # build docs
+        run : pipenv run tests
 
       - name: Coveralls
-        # the official action only uses Lcov files which pytest-cov does not support
-        # https://github.com/coverallsapp/github-action/issues/30
-        #
-        # using https://github.com/coveralls-clients/coveralls-python instead
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           pip install coveralls
           coveralls
 
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout master
+        uses: actions/checkout@v2
+        with:
+          ref: master
+          # require all of history to build all tagged versions' docs
+          fetch-depth: 0
+
+      # also checkout the branch that triggered this build
+      - name: Checkout Source
+        uses: actions/checkout@v2
+
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Install Dependencies
```

Comment:
Is there any way to fork a workflow so we can share the same venv for both docs and test but retain the parallelization?

Reply:
Or at least have YAML references so we don't have to repeat ourselves?

Reply:
I tried to do this and I think at present the answer is no, github actions are a little lacking in this area.

I could use a cache to speed this step but that would require more repetition and several more lines of YAML. 

- separate jobs are in separate VMs so there is no sharing other than via cache
- there does not seem to be any 'include' mechanism